mmv1/products/cloudrun/Service.yaml

```
@@ -53,6 +53,16 @@ examples:
       cloud_run_service_name: 'cloudrun-srv'
     test_env_vars:
       project: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloud_run_service_gpu'
+    primary_resource_id: 'default'
```

```suggestion
    name: 'cloud_run_service_gpu'
    min_version: 'beta'
    primary_resource_id: 'default'
```
We'll need to make the example beta-only as well as it contains beta-only field `node_selector`